docs/examples/ingestion/document_management_pipeline.ipynb

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    documents = SimpleDirectoryReader("./data", filename_as_id=True).load_data()
</pre><p>if there are multi folders like a/1.txt and b/1.txt which both contain the same filename. will it cause issue here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
The ID should have the full filepath actually, but I'll double check